receiver/prometheusreceiver/internal/metrics_adjuster.go

```diff
@@ -289,83 +271,35 @@ func (ma *MetricsAdjuster) adjustPoints(metricType metricspb.MetricDescriptor_Ty
 			zap.Int("len(current)", len(current)), zap.Int("len(initial)", len(initial)), zap.Int("len(previous)", len(previous)))
 		return true
 	}
-	return ma.adjustPoint(metricType, current[0], initial[0], previous[0])
+	return ma.adjustPoint(metricType, current[0], previous[0])
 }
 
-// Note: There is an important, subtle point here. When a new timeseries or a reset is detected,
-// current and initial are the same object. When initial == previous, the previous value/count/sum
-// are all the initial value. When initial != previous, the previous value/count/sum has been
-// adjusted wrt the initial value so both they must be combined to find the actual previous
-// value/count/sum. This happens because the timeseries are updated in-place - if new copies of the
-// timeseries were created instead, previous could be used directly but this would mean reallocating
-// all of the metrics.
 func (ma *MetricsAdjuster) adjustPoint(metricType metricspb.MetricDescriptor_Type,
```

Comment:
nit: can we rename this?  This doesn't do any adjusting of points anymore :).  Maybe `isReset(`?